src/index.native.tsx

```diff
@@ -254,13 +272,19 @@ module.exports = {
   get ScreenStackHeaderSubview() {
     return ScreensNativeModules.NativeScreenStackHeaderSubview;
   },
+  get SearchBar() {
+    if (Platform.OS !== 'ios') {
+      console.warn('Importing SearchBar is only valid on iOS devices.');
+      return View;
+    }
 
-  // these are functions and will not be evaluated until used
-  // so no need to use getters for them
+    return ScreensNativeModules.NativeSearchBar;
```

Comment:
Why is the previous comment removed?

Reply:
Fixed in https://github.com/software-mansion/react-native-screens/pull/758/commits/49247f00af70303e425be0695dba328f3e2b4d4a